src/components/views/messages/RoomPredecessorTile.tsx

```diff
@@ -86,18 +88,32 @@ export const RoomPredecessorTile: React.FC<IProps> = ({ mxEvent, timestamp }) =>
     }
 
     const prevRoom = MatrixClientPeg.get().getRoom(predecessor.roomId);
-    if (!prevRoom) {
+
+    // We need either the previous room, or some servers to find it with.
+    // Otherwise, we must bail out here
+    if (!prevRoom && !predecessor.viaServers) {
         logger.warn(`Failed to find predecessor room with id ${predecessor.roomId}`);
-        return <></>;
-    }
-    const permalinkCreator = new RoomPermalinkCreator(prevRoom, predecessor.roomId);
-    permalinkCreator.load();
-    let predecessorPermalink: string;
-    if (predecessor.eventId) {
-        predecessorPermalink = permalinkCreator.forEvent(predecessor.eventId);
-    } else {
-        predecessorPermalink = permalinkCreator.forRoom();
+        return (
+            <EventTileBubble
+                className="mx_CreateEvent"
+                title={_t("This room is a continuation of another conversation.")}
+                timestamp={timestamp}
+            >
+                <div className="mx_EventTile_body">
+                    <span className="mx_EventTile_tileError">
+                        {_t("Can't find the old version of this room (room id: %(roomId)s).", {
+                            roomId: predecessor.roomId,
+                        })}
+                    </span>
+                </div>
+            </EventTileBubble>
+        );
     }
```

Comment:
It would be good to add a general error tile if `!prevRoom` isn't found but `viaServers` does exist.

The general error message even makes more sense in this case: `"Can't find the old version of this room (room id: %(roomId)s)."`

Reply:
If `prevRoom` isn't found but `viaServers` does exist, I would expect that the room will be found when we navigate to it. That's why I didn't show an error. Am I misunderstanding?

Reply:
I think you're right and I was misunderstanding if this was before or after we tried to navigate to the room 👍

Reply:
We could add a comment explaining this maybe though. "We expect this room to probably be found"

Reply:
Comment added in 7db8b26960